src/views/account/Login.vue

```diff
@@ -36,12 +44,25 @@
                 data: {
                     email: '',
                     password: ''
-                }
+                },
+                loading: false
             }
         },
         methods: {
             login: function () {
                 this.authenticate(this.data)
+            },
+            authenticate: function (data) {
+                this.loading = true;
+                api.account.authenticate(data).then((response) => {
+                        authenticationStore.commit('setToken', response.token)
+                        //@ts-ignore
+                        this.$router.push('/account');
+                    },
+                    error => {
+                        this.loading = false;
+                        this.$router.push('/login')
+                    })
             }
```

Comment:
I think it'd be better to replace the "authenticate" method in account mixin instead of overriding it here. Or to remove it from mixin. Right now we have 2 methods with the same name and signature and I can't really see the profit from it.